cmd/snap-confine/ns-support.c

```diff
@@ -662,6 +718,23 @@ static void helper_capture_ns(struct sc_mount_ns *group, pid_t parent)
 	      (int)parent, dst);
 }
 
+static void helper_capture_per_user_ns(struct sc_mount_ns *group, pid_t parent)
+{
+	char src[PATH_MAX] = { 0 };
+	char dst[PATH_MAX] = { 0 };
+	uid_t uid = getuid();
+
+	debug("capturing per-snap, per-user mount namespace");
+	sc_must_snprintf(src, sizeof src, "/proc/%d/ns/mnt", (int)parent);
+	sc_must_snprintf(dst, sizeof dst, "%s.%d.mnt", group->name, (int)uid);
```

Comment:
To keep with convention, that should probably be:
```
sc_must_snprintf(dst, sizeof dst, "%s.%d%s", group->name, (int)uid, SC_NS_MNT_FILE);
```

Reply:
I'd like to drop that SC_NS_MNT_FILE define. It is not universally used and it clouds the file name more than it helps.

Reply:
Sounds good to me. With the way it's currently defined the printf format string looks a bit awkward.

Reply:
I'll nuke that define.